[RHACS] [Docs] [rhacs-docs-main] ROX-30984: Removing callouts across ACS documentation (#103150)
openshift/openshift-docs: modules/operator-upgrade-modify-central-custom-resource.adoc

spec:
  central:
    db:
      isEnabled: Default
      persistence:
        persistentVolumeClaim:
          claimName: central-db
          size: 100Gi
          storageClassName: <storage-class-name>
----
+
where:
+
--

> `spec.central.db.isEnabled`:: Specifies that you must not change the value of `IsEnabled` to `Enabled`.

Reviewer: not accurate

```suggestion
`spec.central.db.isEnabled`:: Specifies that the database is enabled. Do not change this value from `IsEnabled` to `Enabled`.
```
